examples/pygeosxExamples/sedovWithStressFunction/run_sedov_problem.py

```diff
@@ -18,15 +19,18 @@ def stress_fn(x):
     """
     R = x[:, 0]**2 + x[:, 1]**2 + x[:, 2]**2
     return np.sin(2.0 * np.pi * R / np.amax(R))
+# PYGEOSX_STRESS_FN_END
 
 
 def run_problem():
     """
     Run the GEOSX problem
     """
+    # PYGEOSX_INITIALIZATION
```

Comment:
In this file, `PYGEOSX_INITIALIZATION` encloses the mpi rank stuff while in `hydraulicFractureWithMonitor.py` it is done outside `run_problem`. 
You documentation effort would surely benefit from an homogeneous way of doing things.